guardian/manage-frontend
# Block address update for national delivery

## What does this change?
We currently don't have the capability to allow users with national deliveries to update their addresses via MMA. This change no longer shows the address form and  displays a new message to call the CSR team if you want to change addresses.

This change also prevents users who may have any other delivery as well as a national delivery product from updating their address.



## How to test
On MMA with a national delivery product -> click manage subscription -> click manage delivery address you should see the new message displayed asking you to call a CSR to update your address.

On MMA with a home delivery product (as well a national delivery) -> click manage subscription -> click manage delivery address you should see the new message displayed asking you to call a CSR to update your address.



## How can we measure success?



## Have we considered potential risks?



## Images


National delivery product 
![image](https://github.com/guardian/manage-frontend/assets/115992455/d81ddb4a-eb8d-4101-8a51-a76a83dcf30a)

Home delivery with also an active sub
![image](https://github.com/guardian/manage-frontend/assets/115992455/52fb608a-ba7a-4c35-b228-93623e87ccc0)

Only home delivery on account 🎉
![image](https://github.com/guardian/manage-frontend/assets/115992455/ab6e1a67-0358-4b87-82da-43b0dfd59c63)

## Accessibility
